.github/workflows/publish_commit.yml

```diff
@@ -1,8 +1,12 @@
 name: Publish Commit SHA for performance testing
 on:
-  push:
+  pull_request:
+    types:
+      - closed
```

Comment:
Closed, but not merged? Or does close mean merged here?

Reply:
There is no specific "merged" event. See the documentation, I thought "closed" would be the closest to what we want https://github.com/graphql-java/graphql-java/pull/3867